sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClientBuilder.java

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.okhttp;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.util.logging.ClientLogger;
+import okhttp3.Authenticator;
+import okhttp3.ConnectionPool;
+import okhttp3.Dispatcher;
+import okhttp3.Interceptor;
+import okhttp3.OkHttpClient;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Builder to configure and build an implementation of com.azure.core.http.HttpClient for OkHttp.
+ */
+public class OkHttpAsyncHttpClientBuilder {
+    private final ClientLogger logger = new ClientLogger(OkHttpAsyncHttpClientBuilder.class);
```

Comment:
Looks like the logger is not used.

Reply:
Removed in the [PR](https://github.com/Azure/azure-sdk-for-java/pull/5222).